### PR TITLE
Use ES2015

### PR DESCRIPTION
**Pull request by tymondesigns**
~~Couldn't switch over the tests since `taper` doesn't support it.~~

Did I miss anything else?

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,25 +1,24 @@
...
+const filter = () => {
+  let value = input.value.toLowerCase();
 
-  plugins.forEach(function (plugin) {
-    var el = plugin.el;
-    var title = plugin.title;
-    var desc = plugin.description;
-    plugin.el.style.display = '';
+  plugins.forEach((plugin) => {
+    let { el, title, description } = plugin;
```
Comment (woodyrew) [inline index.js]: I think this could be `const` as they're not being changed.

**Comment by woodyrew** [comment]
I've used es6 syntax with `tape` via `taper`. What problem did you get?

**Comment by tymondesigns** [comment]
Gruntfile done...

It turns out using `import cheerio from 'cheerio';` in the `verify/build.js` file throws an error, but other es6 features such as arrow functions work, so i'll update that now

**Comment by woodyrew** [comment]
Looks great, thanks.

**Comment by tymondesigns** [comment]
👍

**Comment by woodyrew** [comment]
resolves #204

**Comment by woodyrew** [comment]
Resolves #203

**Pull request merged**